validator/testcases/javascript/predefinedentities.py

```diff
@@ -1085,6 +1187,11 @@ def interface_obj(iface):
                             {"value":
                                  {u"enablePrivilege":
                                       {"dangerous": {
+                                          "signing_help":
+                                            "Any references to this API must "
+                                            "be removed from your extension. "
+                                            "Add-ons using this API will not "
+                                            "be accepted for signing.",
```

Comment:
Same as above, not sure `for signing` is useful here (and may even be confusing)